Marlin/ubl_motion.cpp

```diff
@@ -250,9 +231,8 @@
     const float m = dy / dx,
                 c = start[Y_AXIS] - m * start[X_AXIS];
 
-    const bool inf_normalized_flag = isinf(e_normalized_dist),
-               inf_m_flag = isinf(m);
-
+    const bool inf_normalized_flag=isinf(e_normalized_dist),
+               inf_m_flag=isinf(m);
```

Comment:
> if a variable is being declared and it is being assigned a value in the declaration

You understand, this is also not "pre-initialized":
```cpp
int x = 5;
```
It's just a syntactical combining the declaration and assignment. It is functionally identical to:
```cpp
int x;
x = 5;
```